client/files.go

```diff
@@ -364,3 +368,135 @@ func (client *Client) RemovePath(opts *RemovePathOptions) error {
 
 	return nil
 }
+
+// PushOptions contains the options for a call to Push.
```

Comment:
Nit: we've moved towards not adding doc comments for "obvious" types/functions, like the comment for `PushOptions` basically saying "options for Push". So let's remove this.